prd.md

```
@@ -0,0 +1,206 @@
+
+### Arquivos a Modificar
+
+| Arquivo | Mudança |
+|---|---|
+| `app/javascript/dashboard/routes/dashboard/conversation/scheduledMessages/ScheduledMessageModal.vue` | Substituir seção do DatePicker por `<ScheduleDateShortcuts v-model="scheduledDateTime" />`. Remover imports e funções relacionadas ao DatePicker que migram para o novo componente |
+| `app/javascript/dashboard/i18n/locale/en/conversation.json` | Adicionar chaves `SCHEDULED_MESSAGES.MODAL.SHORTCUTS.*` |
+| `app/javascript/dashboard/i18n/locale/pt_BR/conversation.json` | Adicionar traduções pt-BR correspondentes |
+
+### Fluxo de Dados
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)